doc/installation-guide.md

```diff
@@ -79,12 +79,13 @@ The `CMakeLists.txt` from `iceoryx_meta` can be used to easily develop iceoryx w
 
  |  switch  |  description |
  |:---------|:-------------|
- | `IOX_MAX_PORT_NUMBER` | the maximum number of publisher and subscriber ports `RouDi` can distribute to the clients |
+ | `IOX_MAX_PUBLISHERS` | the maximum number of publishers one `RouDi` instance can manage |
```

Comment:
while you're at it, you could also make MAX_PROCESS_NUMBER a compile time option

Reply:
OK, I will have to introduce new ones fro clients and servers anyway. But in a next PR, will put it on my list